src/renderer/components/Map.tsx

```diff
@@ -62,17 +64,22 @@ function makeStationMarker(
   const diameter = circleDiameter(species.size(), maxSpecies);
   const maxPreviewPhotosCount = 3;
 
-  const markerElement = document.createElement('div');
-  markerElement.className = styles.marker;
-  markerElement.setAttribute('style', `width: ${diameter}px; height: ${diameter}px;`);
+  const markerElement = document.createElement('img');
+  markerElement.setAttribute('src', Marker);
+
+  // Change marker color when clicked
   if (markerElement) {
     markerElement.addEventListener('click', (event: Event) => {
-      const activeMarkers: NodeListOf<Element> = document.querySelectorAll(
-        `.${styles.marker}.${styles.active}`
-      );
-      activeMarkers.forEach((marker) => marker.classList.remove(styles.active));
+      markerElement.setAttribute('src', MarkerActive);
+      markerElement.className = '.active-marker';
```

Comment:
Do we need to set the `.active-marker`? It used to control the appearance of the markers but it seems to do nothing now.

Reply:
I needed a way to select clicked markers so I added a className of ".active-marker" to markers being clicked.